rs/ledger_suite/icrc1/ledger/src/tests.rs

```diff
@@ -108,7 +126,7 @@ fn test_approvals_are_not_cumulative() {
         Allowance {
             amount: approved_amount,
             expires_at: None,
-            arrived_at: now,
+            arrived_at: ts(0),
```

Comment:
Why are we testing with 0 as opposed to some arbitrary non-zero value (making sure to distinguish 0 as the default value of the integer type and the actual value for the sake of the test)?

Reply:
`StorableAllowance` always sets `arrived_at` to 0 so the assert would fail with any other value. We have to keep the field for now as it is still used by the ICP ledger. Do you suggest to use a different value for the unused field?